content/en/admin/optional/tor.md

```diff
@@ -187,6 +187,8 @@ An example nginx configuration would look like (replace the onion name with your
 add_header Alt-Svc 'h2="qKnF…sKq7.onion:443"; ma=86400; persist=1';
 ```
 
+This example uses port 443, but that is not required and could be configured differently. This configuration allows HTTPS to be used with a normal non-onion TLS certificate.
```

Comment:
This incorrectly implies that using port 443 is what allows it to use the normal TLS certificate.

The reason a normal certificate can be used is that the TLS connection doesn't terminate at the onion service. In fact, the onion service could be a completely different machine than the server it directs traffic to (though this would obviously introduce more latency) without leaking unencrypted traffic at any point.